common4j/src/main/com/microsoft/identity/common/java/commands/parameters/nativeauth/ResetPasswordSubmitCodeCommandParameters.java

```diff
@@ -0,0 +1,50 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.common.java.commands.parameters.nativeauth;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.experimental.SuperBuilder;
+
+/**
+ * A set of Reset Password Submit Code command parameters for submitting the one-time password to the server for authentication.
+ * extends from {@link BaseNativeAuthCommandParameters
+ */
+@Getter
+@EqualsAndHashCode(callSuper = true)
+@SuperBuilder(toBuilder = true)
+public class ResetPasswordSubmitCodeCommandParameters extends BaseNativeAuthCommandParameters {
+
+    /**
+     * The one-time password used for authentication.
+     */
+    @NonNull
+    public final String code;
+
+    /**
+     * The password reset token obtained from the challenge endpoint.
```

Comment:
For all of these tokens can we add in the javadoc that these are flow token?

Perhaps even rename like `passwordResetFlowToken`?

Reply:
The API team have decided to rename all of these tokens as continuation_token. We are preparing a PR in parallel for that work. As soon as this PR is merged, we will create a new PR to rename all tokens to continuation_token.